controllers/gslb_controller.go

```diff
@@ -56,6 +56,7 @@ type GslbReconciler struct {
 
 const (
 	gslbFinalizer                        = "k8gb.absa.oss/finalizer"
+	geoStrategy                          = "geo"
```

Comment:
@k0da Should it be rather "geoip", to be more precise? This strategy is widely known as GeoIP strategy, not Geo strategy.
```suggestion
	geoStrategy                          = "geoip"
```